charts/csm-authorization/templates/csm-config-params.yaml

```diff
@@ -7,6 +7,7 @@ data:
   csm-config-params.yaml: |
     CONCURRENT_POWERFLEX_REQUESTS: {{ .Values.authorization.concurrentPowerFlexRequests }}
     LOG_LEVEL: {{ .Values.authorization.logLevel }}
+    storageCapacityPollInterval: {{ .Values.authorization.storageCapacityPollInterval }}
```

Comment:
Can we capitalize this to be consistent with CONCURRENT_POWERFLEX_REQUESTS and LOG_LEVEL?

Reply:
Sure, fixed. Thank you!